### PR TITLE
Linux ARM Support for Devices like the Raspberry Pi 3

### PR DESCRIPTION
**Pull request by dayyan**
Ombi running on a Raspberry Pi 3.
The .NET Core SDK [isn't avaliable on the Pi](https://github.com/dotnet/core/blob/master/samples/RaspberryPiInstructions.md). You must build elsewhere, and target it.

Steps to Run:
1. Install .NET core with [these instructions](https://blogs.msdn.microsoft.com/david/2017/07/20/setting_up_raspian_and_dotnet_core_2_0_on_a_raspberry_pi/).
2. Follow [V3 installation instructions](https://github.com/tidusjar/Ombi/wiki/Installation).

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -204,6 +208,17 @@ Task("Publish-Linux")
...
 });
 
+Task("Publish-Linux-ARM")
+    .Does(() =>
+{
+    publishSettings.Runtime = "linux-arm";
+    publishSettings.OutputDirectory = Directory(buildDir) + Directory("netcoreapp2.0/linux-arm/published");
+
+    DotNetCorePublish("./src/Ombi/Ombi.csproj", publishSettings);
+    CopyFile(buildDir + "/netcoreapp2.0/linux-x64/Swagger.xml", buildDir + "/netcoreapp2.0/linux-arm/published/Swagger.xml");
```
Comment (tidusjar) [inline build.cake]: `/netcoreapp2.0/linux-x64/Swagger.xml` ? Shouldn't this be `/linux-arm/`?

File: src/Ombi.Mapping/Ombi.Mapping.csproj [hunk 2]
```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="AutoMapper" Version="6.1.1" />
     <PackageReference Include="AutoMapper.Collection" Version="3.1.2" />
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="3.0.1" />
+    <PackageReference Include="SQLitePCLRaw.bundle_e_sqlite3" Version="1.1.9" />
```
Comment (tidusjar) [inline src/Ombi.Mapping/Ombi.Mapping.csproj]: Why has this been added?

Comment (dayyan) [reply src/Ombi.Mapping/Ombi.Mapping.csproj]: While running the ARM build I ran into an issue similar to this: https://github.com/ericsink/SQLitePCL.raw/issues/161

```Unhandled Exception: System.TypeInitializationException: The type initializer for 'Microsoft.Data.Sqlite.SqliteConnection' threw an exception. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.DllNotFoundException: Unable to load DLL 'e_sqlite3': The specified module could not be found.```

Adding this solved it. I haven't dabbled in .NET in a long time, so if there's a better way to solve that issue...

File: build.cake (hunk 1 above)
Comment (dayyan) [reply build.cake]: Will do.

**Comment by dayyan** [comment]
Linking, https://github.com/tidusjar/Ombi/issues/1716

File: src/Ombi.Mapping/Ombi.Mapping.csproj (hunk 2 above)
Comment (tidusjar) [reply src/Ombi.Mapping/Ombi.Mapping.csproj]: sorry for the long reply, this is a strange project to include this reference, you 100% sure you meant for it to be in this project and not Ombi.Store?

**Comment by tidusjar** [comment]
I need to modify the automatic updater and also my external service to also support this. This is why I have not merged yet.

File: src/Ombi.Mapping/Ombi.Mapping.csproj (hunk 2 above)
Comment (dayyan) [reply src/Ombi.Mapping/Ombi.Mapping.csproj]: I'll try that, traveling right now.

**Comment by dayyan** [comment]
🎉

**Pull request merged**